master.yaml

```diff
@@ -16,7 +16,7 @@ Description: >
     Based on AWSLabs ECS Reference Arhitecture
     https://github.com/awslabs/ecs-refarch-cloudformation
 
-    Last Modified: 17th February 2017
+    Last Modified: 13th May 2017
```

Comment:
2018

Reply:
OMG it's one year to the *day* since Dan or I made this update.  Freaky.  And yes, probably good to note the update to *now* that I'm imposing back to the repo (though I was oblivious to anything but the _historical_ significance of this date field).